.github/workflows/integrationTests.yaml

```diff
@@ -118,3 +118,111 @@ jobs:
           path: |
             **/target/failsafe-reports/*
             **/target/site
+
+  spring-boot-3-2-compatibility:
+    if: |
+      github.actor != 'dependabot[bot]' && ((
+        github.event_name == 'pull_request' && github.repository == github.event.pull_request.head.repo.full_name
+      ) || (github.event_name != 'pull_request'))
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        it:
+          - alloydb
+          - bigquery
+          - cloudsql
+          - config
+          - datastore
+          - firestore
+          - kms
+          - kotlin
+          - logging
+          - metrics
+          - multisample
+          - pubsub
+          - pubsub-bus
+          - pubsub-docs
+          - pubsub-emulator
+          - pubsub-integration
+          - secretmanager
+          - spanner
+          - storage
+          # trace excluded
```

Comment:
Should we include it, but explicitly skip it so it still shows up, but as skipped?

Reply:
I'm guessing the conditional execution check occurs at the job level, not the individual strategy level. If so, the strategy variable might not be in scope... but giving it a try.